ansible/roles/freeipa/tasks/users.yml

```diff
@@ -0,0 +1,4 @@
+- name: Add users to freeipa
+  # This uses DNS to find the ipa server, which works as this is running on the enrolled ipa server
+  community.general.ipa_user: "{{ freeipa_user_defaults | combine(item) }}"
```

Comment:
Splatting args like this causes a warning - any chance you could split out the arguments that you want to use?